tests/test_yamls/test_storage_mounting.yaml

```diff
@@ -0,0 +1,45 @@
+file_mounts:
+  # Mounting public buckets
+  /mount_public_s3:
+    source: s3://digitalcorpora
+    mode: MOUNT
+
+  # Mounting public buckets
+  /mount_public_gcp:
+    source: gs://gcp-public-data-sentinel-2
+    mode: MOUNT
+
+  # Mounting private buckets in COPY mode
+  /mount_private_copy:
+    name: random_bucket_name
+    source: ~/tmp-workdir
+    mode: COPY
+
+  # Mounting private buckets in MOUNT mode
+  /mount_private_mount:
+    name: random_bucket_name
+    source: ~/tmp-workdir
```

Comment:
Just curious, since the same source is specified twice to the same bucket, there is redundant uploading done? Should we allow for users to do this? (assert unique?)

Reply:
Yeah, it would we uploaded twice... we should do some optimizations for this in storage.

Reply:
Sg, ig lets keep it separate from this PR then